scripts/Dockerfile.bottlecap.build

```diff
@@ -6,12 +6,14 @@ RUN yum install -y curl gcc gcc-c++ make unzip openssl openssl-devel
 COPY ./scripts/install-protoc.sh /
 RUN chmod +x /install-protoc.sh && /install-protoc.sh
 RUN curl https://sh.rustup.rs -sSf | \
-    sh -s -- --default-toolchain stable -y
+    sh -s -- --default-toolchain nightly-x86_64-unknown-linux-gnu -y
```

Comment:
This has to be changed so `ARCHITECTURE` passed through the build process is honored.

Reply:
Done